### PR TITLE
separate wielder skills into "skills" and "startingskills"

### PR DESCRIPTION
**Pull request by Unei95**
- `convert.mjs` now separates skills into `startingSkills` and `skills`
- added `WielderDTO.startingSkills` field
- render startingSkills in own table in `pages/wielders/[type].tsx`
- removed "Command" from `WielderStats.tsx` since it's shown in the startingSkills

### CODE REVIEW

File: pages/wielders/[type].tsx [hunk 1]
```diff
@@ -211,6 +233,7 @@ export const getStaticProps = withStaticBase(async (context) => {
     commandDescription: getTerm("Skills/Command/Level3/Description", locale),
     and: getSiteTerm("And", locale),
     or: getSiteTerm("Or", locale),
+    startingSkills: getSiteTerm("StartingSkills", locale),
```
Comment (lmachens) [inline pages/wielders/[type].tsx:236]: I think you could remove L:208 (I can't add a line-comment there) :
`command: getTerm("Commanders/Details/CommanderStat/Command", locale),`

**Review by lmachens — CHANGES_REQUESTED** [review]
\^o^/
Good quality pull request!

It looks like you converted not the most recent Songs of Conquest assets.
Could you update SoC and run the extractor again?

File: pages/wielders/[type].tsx [hunk 2]
```diff
@@ -151,7 +173,7 @@ const Wielder: NextPage<{ wielder: WielderDTO; icons: IconsDTO }> = ({
                             {requiredSkill.name}
                           </Text>
                         </PopoverLink>
-                        {}
+                        { }
```
[the remark this answers is not in the document]
Comment (Unei95) [reply pages/wielders/[type].tsx]: resolved in 79b2bd13ed1c75214464a55911125107442703da

**Comment by Unei95** [comment]
i forgot to run assetRipper prior to running the extractor

facepalm

**Review by lmachens — APPROVED** [review]
Good job!
For some reason, the binary data of GOG images are different from the Steam images (or is it related to Linux?).  I pushed the steam images, because they didn't change.

**Pull request merged**